cypress/support/step_definitions/dynamicCalls.md

```diff
@@ -1,7 +1,9 @@
 # DynamicCallsGlue
 
+## [Dynamic Firebolt Objects](../../fixtures/docs/dynamicObjects.md)
```

Comment:
table of content to be added

Dynamic glue codes have to be used with dynamic objects. Details on dynamic objects here (link)

Reply:
done